dashboard/Makefile

```diff
@@ -1,8 +1,11 @@
 GOCMD ?= go
 GOARGS = -trimpath
 CONFIG_TEMPLATE_DIR ?= config
+PSM ?= false
```

Comment:
I would just make PSM ?= with nothing.

Then you can use conditions all the way like this:

ifeq ($(PSM),)
whatever you want
else
something else
endif

In particular, only set PSM_DASHBOARDS_CONFIG_DIR if PSM is set, and set REPLICATOR_CONFIG_TEMPLATE depending on whether PSM is set (no need for PSM version).

If you are feeling adventurous you can just add the PSM version of the dashboard location to the regular variable and then use the cp command directly with the two variables.